app/src/main/java/org/coepi/android/ui/extensions/ProgressBarViewExtension.kt

```diff
@@ -0,0 +1,24 @@
+package org.coepi.android.ui.extensions
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import org.coepi.android.R
+
+
+class ProgressBarViewExtension : LinearLayout {
+
+    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {
+        orientation = HORIZONTAL
```

Comment:
If we do need a custom view, I think that you can use `FrameLayout`, which is more lightweight. You can use `layout_gravity` in the inner element to center. But not critical.